Classes/ViewHelpers/Field/TextViewHelper.php

```diff
@@ -65,6 +65,15 @@ public function initializeArguments()
             false,
             ''
         );
+        $this->registerArgument(
+            'richtextConfiguration',
+            'string',
+            'Specifies which configuration to use in combination with EXT:rte_ckeditor.' .
+            'The default value is \'default\', since that\'s the standard configuration the CMS uses. ' .
+            'More information: https://docs.typo3.org/typo3cms/TCAReference/ColumnsConfig/Properties/TextRichtextConfiugration.html',
+            false,
+            null
```

Comment:
The default value here should be the value of the constant (either the newly added class constant or the one TYPO3 contains, if it contains one).